src/data/textures.c

```diff
@@ -2045,404 +2045,404 @@ MkAnimation D_0200210C[2] = {
  * @brief unknown MenuTexture
  *
  */
-MenuTexture D_0200211C[2] = {
-    { 5, gTexture7F2094, 26, 16, 0, 0, 0x0, 0 },
+MenuTexture gTextureFontLetterA[2] = {
```

Comment:
This segment is not global. It's segment 2

seg2_texture_font_letter_a
or
seg2_textureFontLetterA